pkg/nimble/Makefile

```diff
@@ -21,7 +21,8 @@ else
   CFLAGS += -Wno-unused-but-set-variable
 endif
 
-SUBMODS := $(filter nimble_%,$(USEMODULE))
+IGNORE := nimble_autoconn_%
+SUBMODS := $(filter-out $(IGNORE),$(filter nimble_%,$(USEMODULE)))
```

Comment:
Is this something like `default` and `non-default` sub-modules? Can you clarify?

Reply:
nope, its simply the pseudomodule adaption for the nimble package.